iib/workers/config.py

```diff
@@ -74,6 +80,19 @@ def validate_celery_config(conf, **kwargs):
     Perform basic validatation on the Celery configuration when the worker is initialized.
 
     :param celery.app.utils.Settings conf: the Celery application configuration to validate
+    :raises iib.exceptions.ConfigError: if the configuration is invalid
     """
-    # For now, don't perform any validation
-    pass
+    if not conf.get('iib_registry'):
+        raise ConfigError('iib_registry must be set to the destination container registry')
+
+    if not conf.get('iib_registry_credentials'):
+        raise ConfigError(
+            'iib_registry_credentials must be set in the format of "username:password"'
+        )
+
+
+def get_worker_config():
+    # Import this here to avoid a circular import
+    import iib.workers.tasks.celery
+
```

Comment:
optional: this newline seems unnecessary. we could remove it.

Reply:
I've been using the auto-formatting tool "black" and it did this automatically, so I'll keep it as is.